SDL master FTBFS: XTestQueryExtension undeclared

Platform: openSUSE Tumbleweed 20250515 amd64
Commit 4cd889cb1313fdb1c7f0904f7d23cd1b4b4d2e2c:

cmake line

<details>

```
$ /usr/bin/cmake . '-GUnix Makefiles' -DCMAKE_INSTALL_PREFIX:PATH=/usr -DCMAKE_INSTALL_BINDIR:PATH=bin -DCMAKE_INSTALL_SBINDIR:PATH=sbin -DCMAKE_INSTALL_LIBEXECDIR:PATH=libexec -DCMAKE_INSTALL_SYSCONFDIR:PATH=etc -DCMAKE_INSTALL_SHAREDSTATEDIR:PATH=/var/lib -DCMAKE_INSTALL_LOCALSTATEDIR:PATH=var -DCMAKE_INSTALL_RUNSTATEDIR:PATH=run -DCMAKE_INSTALL_LIBDIR:PATH=lib64 -DCMAKE_INSTALL_INCLUDEDIR:PATH=include -DCMAKE_INSTALL_DATAROOTDIR:PATH=share -DCMAKE_INSTALL_DOCDIR:PATH=share/doc/packages/SDL3 -DCMAKE_INSTALL_MANDIR:PATH=share/man -DCMAKE_INSTALL_INFODIR:PATH=share/info -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DCMAKE_BUILD_TYPE=RelWithDebInfo '-DCMAKE_C_FLAGS=-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -ffat-lto-objects -g' '-DCMAKE_CXX_FLAGS=-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -ffat-lto-objects -g' '-DCMAKE_Fortran_FLAGS=-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -ffat-lto-objects -g' '-DCMAKE_EXE_LINKER_FLAGS=-flto=auto -ffat-lto-objects -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' '-DCMAKE_MODULE_LINKER_FLAGS=-flto=auto -ffat-lto-objects -Wl,--as-needed' '-DCMAKE_SHARED_LINKER_FLAGS=-flto=auto -ffat-lto-objects -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' -DLIB_SUFFIX=64 -DCMAKE_VERBOSE_MAKEFILE:BOOL=OFF -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_LIBS:BOOL=OFF -DCMAKE_COLOR_MAKEFILE:BOOL=ON -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF -DCMAKE_MODULES_INSTALL_DIR=/usr/lib64/cmake/SDL3 -DSDL_SSE3:BOOL=OFF -DSDL_3DNOW:BOOL=OFF -DSDL_ALSA_SHARED:BOOL=OFF -DSDL_LIBSAMPLERATE_SHARED:BOOL=OFF -DSDL_PIPEWIRE_SHARED:BOOL=OFF -DSDL_PULSEAUDIO_SHARED:BOOL=OFF -DSDL_X11_SHARED:BOOL=OFF -DSDL_WAYLAND_SHARED:BOOL=OFF -DSDL_KMSDRM_SHARED:BOOL=OFF -DSDL_STATIC:BOOL=OFF -DSDL_STATIC_PIC:BOOL=ON -DSDL_RPATH:BOOL=OFF -DSDL_TEST_LIBRARY:BOOL=OFF -DSDL_INSTALL_DOCS:BOOL=ON
```

</details>

cmake output:

<details>

```
-- The C compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.2.0")
-- Detecting Target CPU Architecture
-- Detecting Target CPU Architecture - X64
-- Performing Test HAVE_GCC_WALL
-- Performing Test HAVE_GCC_WALL - Success
-- Performing Test HAVE_GCC_WUNDEF
-- Performing Test HAVE_GCC_WUNDEF - Success
-- Performing Test HAVE_GCC_WFLOAT_CONVERSION
-- Performing Test HAVE_GCC_WFLOAT_CONVERSION - Success
-- Performing Test HAVE_GCC_NO_STRICT_ALIASING
-- Performing Test HAVE_GCC_NO_STRICT_ALIASING - Success
-- Performing Test HAVE_GCC_WDOCUMENTATION
-- Performing Test HAVE_GCC_WDOCUMENTATION - Failed
-- Performing Test HAVE_GCC_WDOCUMENTATION_UNKNOWN_COMMAND
-- Performing Test HAVE_GCC_WDOCUMENTATION_UNKNOWN_COMMAND - Failed
-- Performing Test HAVE_GCC_COMMENT_BLOCK_COMMANDS
-- Performing Test HAVE_GCC_COMMENT_BLOCK_COMMANDS - Failed
-- Performing Test HAVE_CLANG_COMMENT_BLOCK_COMMANDS
-- Performing Test HAVE_CLANG_COMMENT_BLOCK_COMMANDS - Failed
-- Performing Test HAVE_GCC_WSHADOW
-- Performing Test HAVE_GCC_WSHADOW - Success
-- Performing Test HAVE_GCC_WUNUSED_LOCAL_TYPEDEFS
-- Performing Test HAVE_GCC_WUNUSED_LOCAL_TYPEDEFS - Success
-- Performing Test HAVE_GCC_WIMPLICIT_FALLTHROUGH
-- Performing Test HAVE_GCC_WIMPLICIT_FALLTHROUGH - Success
-- Performing Test COMPILER_SUPPORTS_FDIAGNOSTICS_COLOR_ALWAYS
-- Performing Test COMPILER_SUPPORTS_FDIAGNOSTICS_COLOR_ALWAYS - Success
-- Looking for __GLIBC__
-- Looking for __GLIBC__ - found
-- Performing Test LINKER_SUPPORTS_VERSION_SCRIPT
-- Performing Test LINKER_SUPPORTS_VERSION_SCRIPT - Success
-- Performing Test COMPILER_SUPPORTS_GCC_ATOMICS
-- Performing Test COMPILER_SUPPORTS_GCC_ATOMICS - Success
-- Performing Test HAVE_GCC_FVISIBILITY
-- Performing Test HAVE_GCC_FVISIBILITY - Success
-- Performing Test LINKER_SUPPORTS_WL_NO_UNDEFINED
-- Performing Test LINKER_SUPPORTS_WL_NO_UNDEFINED - Success
-- Performing Test COMPILER_SUPPORTS_MMX
-- Performing Test COMPILER_SUPPORTS_MMX - Success
-- Performing Test COMPILER_SUPPORTS_SSE
-- Performing Test COMPILER_SUPPORTS_SSE - Success
-- Performing Test COMPILER_SUPPORTS_SSE2
-- Performing Test COMPILER_SUPPORTS_SSE2 - Success
-- Performing Test COMPILER_SUPPORTS_SSE4_1
-- Performing Test COMPILER_SUPPORTS_SSE4_1 - Success
-- Performing Test COMPILER_SUPPORTS_SSE4_2
-- Performing Test COMPILER_SUPPORTS_SSE4_2 - Success
-- Performing Test COMPILER_SUPPORTS_AVX
-- Performing Test COMPILER_SUPPORTS_AVX - Success
-- Performing Test COMPILER_SUPPORTS_AVX2
-- Performing Test COMPILER_SUPPORTS_AVX2 - Success
-- Performing Test COMPILER_SUPPORTS_AVX512F
-- Performing Test COMPILER_SUPPORTS_AVX512F - Success
-- Looking for alloca.h
-- Looking for alloca.h - found
-- Looking for float.h
-- Looking for float.h - found
-- Looking for iconv.h
-- Looking for iconv.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for malloc.h
-- Looking for malloc.h - found
-- Looking for math.h
-- Looking for math.h - found
-- Looking for memory.h
-- Looking for memory.h - found
-- Looking for signal.h
-- Looking for signal.h - found
-- Looking for stdarg.h
-- Looking for stdarg.h - found
-- Looking for stdbool.h
-- Looking for stdbool.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stdio.h
-- Looking for stdio.h - found
-- Looking for stdlib.h
-- Looking for stdlib.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for time.h
-- Looking for time.h - found
-- Looking for wchar.h
-- Looking for wchar.h - found
-- Looking for pow in m
-- Looking for pow in m - found
-- Looking for abs
-- Looking for abs - found
-- Looking for acos
-- Looking for acos - found
-- Looking for acosf
-- Looking for acosf - found
-- Looking for asin
-- Looking for asin - found
-- Looking for asinf
-- Looking for asinf - found
-- Looking for atan
-- Looking for atan - found
-- Looking for atan2
-- Looking for atan2 - found
-- Looking for atan2f
-- Looking for atan2f - found
-- Looking for atanf
-- Looking for atanf - found
-- Looking for atof
-- Looking for atof - found
-- Looking for atoi
-- Looking for atoi - found
-- Looking for bcopy
-- Looking for bcopy - found
-- Looking for ceil
-- Looking for ceil - found
-- Looking for ceilf
-- Looking for ceilf - found
-- Looking for copysign
-- Looking for copysign - found
-- Looking for copysignf
-- Looking for copysignf - found
-- Looking for cos
-- Looking for cos - found
-- Looking for cosf
-- Looking for cosf - found
-- Looking for _Exit
-- Looking for _Exit - found
-- Looking for exp
-- Looking for exp - found
-- Looking for expf
-- Looking for expf - found
-- Looking for fabs
-- Looking for fabs - found
-- Looking for fabsf
-- Looking for fabsf - found
-- Looking for floor
-- Looking for floor - found
-- Looking for floorf
-- Looking for floorf - found
-- Looking for fmod
-- Looking for fmod - found
-- Looking for fmodf
-- Looking for fmodf - found
-- Looking for fopen64
-- Looking for fopen64 - found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for fseeko64
-- Looking for fseeko64 - found
-- Looking for getenv
-- Looking for getenv - found
-- Looking for _i64toa
-- Looking for _i64toa - not found
-- Looking for index
-- Looking for index - found
-- Looking for itoa
-- Looking for itoa - not found
-- Looking for log
-- Looking for log - found
-- Looking for log10
-- Looking for log10 - found
-- Looking for log10f
-- Looking for log10f - found
-- Looking for logf
-- Looking for logf - found
-- Looking for lround
-- Looking for lround - found
-- Looking for lroundf
-- Looking for lroundf - found
-- Looking for _ltoa
-- Looking for _ltoa - not found
-- Looking for malloc
-- Looking for malloc - found
-- Looking for memcmp
-- Looking for memcmp - found
-- Looking for memcpy
-- Looking for memcpy - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for memset
-- Looking for memset - found
-- Looking for modf
-- Looking for modf - found
-- Looking for modff
-- Looking for modff - found
-- Looking for pow
-- Looking for pow - found
-- Looking for powf
-- Looking for powf - found
-- Looking for putenv
-- Looking for putenv - found
-- Looking for rindex
-- Looking for rindex - found
-- Looking for round
-- Looking for round - found
-- Looking for roundf
-- Looking for roundf - found
-- Looking for scalbn
-- Looking for scalbn - found
-- Looking for scalbnf
-- Looking for scalbnf - found
-- Looking for setenv
-- Looking for setenv - found
-- Looking for sin
-- Looking for sin - found
-- Looking for sinf
-- Looking for sinf - found
-- Looking for sqr
-- Looking for sqr - not found
-- Looking for sqrt
-- Looking for sqrt - found
-- Looking for sqrtf
-- Looking for sqrtf - found
-- Looking for sscanf
-- Looking for sscanf - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strcmp
-- Looking for strcmp - found
-- Looking for strlcat
-- Looking for strlcat - found
-- Looking for strlcpy
-- Looking for strlcpy - found
-- Looking for strlen
-- Looking for strlen - found
-- Looking for strncmp
-- Looking for strncmp - found
-- Looking for strnlen
-- Looking for strnlen - found
-- Looking for strpbrk
-- Looking for strpbrk - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strnstr
-- Looking for strnstr - not found
-- Looking for strtod
-- Looking for strtod - found
-- Looking for strtok_r
-- Looking for strtok_r - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoll
-- Looking for strtoll - found
-- Looking for strtoul
-- Looking for strtoul - found
-- Looking for strtoull
-- Looking for strtoull - found
-- Looking for tan
-- Looking for tan - found
-- Looking for tanf
-- Looking for tanf - found
-- Looking for trunc
-- Looking for trunc - found
-- Looking for truncf
-- Looking for truncf - found
-- Looking for unsetenv
-- Looking for unsetenv - found
-- Looking for vsnprintf
-- Looking for vsnprintf - found
-- Looking for vsscanf
-- Looking for vsscanf - found
-- Looking for wcsnlen
-- Looking for wcsnlen - found
-- Looking for wcscmp
-- Looking for wcscmp - found
-- Looking for wcsdup
-- Looking for wcsdup - found
-- Looking for wcslcat
-- Looking for wcslcat - found
-- Looking for wcslcpy
-- Looking for wcslcpy - found
-- Looking for wcslen
-- Looking for wcslen - found
-- Looking for wcsncmp
-- Looking for wcsncmp - found
-- Looking for wcsstr
-- Looking for wcsstr - found
-- Looking for wcstol
-- Looking for wcstol - found
-- Looking for strcasestr
-- Looking for strcasestr - found
-- Performing Test LIBC_HAS_ISINF
-- Performing Test LIBC_HAS_ISINF - Success
-- Performing Test LIBC_ISINF_HANDLES_FLOAT
-- Performing Test LIBC_ISINF_HANDLES_FLOAT - Success
-- Performing Test LIBC_HAS_ISINFF
-- Performing Test LIBC_HAS_ISINFF - Success
-- Performing Test LIBC_HAS_ISNAN
-- Performing Test LIBC_HAS_ISNAN - Success
-- Performing Test LIBC_ISNAN_HANDLES_FLOAT
-- Performing Test LIBC_ISNAN_HANDLES_FLOAT - Success
-- Performing Test LIBC_HAS_ISNANF
-- Performing Test LIBC_HAS_ISNANF - Success
-- Looking for fdatasync
-- Looking for fdatasync - found
-- Looking for gethostname
-- Looking for gethostname - found
-- Looking for getpagesize
-- Looking for getpagesize - found
-- Looking for sigaction
-- Looking for sigaction - found
-- Looking for setjmp
-- Looking for setjmp - found
-- Looking for nanosleep
-- Looking for nanosleep - found
-- Looking for gmtime_r
-- Looking for gmtime_r - found
-- Looking for localtime_r
-- Looking for localtime_r - found
-- Looking for nl_langinfo
-- Looking for nl_langinfo - found
-- Looking for sysconf
-- Looking for sysconf - found
-- Looking for sysctlbyname
-- Looking for sysctlbyname - not found
-- Looking for getauxval
-- Looking for getauxval - found
-- Looking for elf_aux_info
-- Looking for elf_aux_info - not found
-- Looking for poll
-- Looking for poll - found
-- Looking for memfd_create
-- Looking for memfd_create - found
-- Looking for posix_fallocate
-- Looking for posix_fallocate - found
-- Looking for posix_spawn_file_actions_addchdir
-- Looking for posix_spawn_file_actions_addchdir - not found
-- Looking for posix_spawn_file_actions_addchdir_np
-- Looking for posix_spawn_file_actions_addchdir_np - found
-- Performing Test ICONV_IN_LIBC
-- Performing Test ICONV_IN_LIBC - Success
-- Performing Test ICONV_IN_LIBICONV
-- Performing Test ICONV_IN_LIBICONV - Failed
-- Performing Test HAVE_SA_SIGACTION
-- Performing Test HAVE_SA_SIGACTION - Success
-- Performing Test HAVE_ST_MTIM
-- Performing Test HAVE_ST_MTIM - Success
-- Looking for dlopen
-- Looking for dlopen - found
-- Performing Test HAVE_O_CLOEXEC
-- Performing Test HAVE_O_CLOEXEC - Success
-- Found ALSA: /usr/lib64/libasound.so (found version "1.2.14")
-- Checking for module 'jack'
--   Found jack, version 1.9.22
-- dynamic libjack -> libjack.so.0
-- Checking for module 'libpipewire-0.3>=0.3.44'
--   Found libpipewire-0.3, version 1.4.2
-- dynamic libpipewire-0.3 -> libpipewire-0.3.so.0
-- Checking for module 'libpulse>=0.9.15'
--   Found libpulse, version 17.0
-- dynamic libpulse -> libpulse.so.0
-- Checking for module 'sndio'
--   Package 'sndio' not found
-- Found X11: /usr/include
-- Looking for XOpenDisplay in /usr/lib64/libX11.so;/usr/lib64/libXext.so
-- Looking for XOpenDisplay in /usr/lib64/libX11.so;/usr/lib64/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- dynamic libX11 -> libX11.so.6
-- dynamic libXext -> libXext.so.6
-- dynamic libXcursor -> libXcursor.so.1
-- dynamic libXi -> libXi.so.6
-- dynamic libXfixes -> libXfixes.so.3
-- dynamic libXrandr -> libXrandr.so.2
-- dynamic libXrender -> libXrender.so.1
-- dynamic libXss -> libXss.so.1
-- dynamic libXtst -> libXtst.so.6
-- Looking for shmat
-- Looking for shmat - found
-- Performing Test HAVE_XGENERICEVENT
-- Performing Test HAVE_XGENERICEVENT - Success
-- Looking for XkbLookupKeySym
-- Looking for XkbLookupKeySym - found
-- Performing Test HAVE_XINPUT2_MULTITOUCH
-- Performing Test HAVE_XINPUT2_MULTITOUCH - Success
-- Performing Test HAVE_XFIXES_H
-- Performing Test HAVE_XFIXES_H - Success
-- Found OpenGL: /usr/lib64/libOpenGL.so
-- Performing Test HAVE_OPENGL_EGL
-- Performing Test HAVE_OPENGL_EGL - Success
-- Checking for module 'libdrm'
--   Found libdrm, version 2.4.124
-- Checking for module 'gbm'
--   Found gbm, version 25.0.5
-- Performing Test HAVE_OPENGL_GLX
-- Performing Test HAVE_OPENGL_GLX - Success
-- Performing Test HAVE_OPENGL
-- Performing Test HAVE_OPENGL - Success
-- Performing Test HAVE_OPENGLES_V1
-- Performing Test HAVE_OPENGLES_V1 - Success
-- Performing Test HAVE_OPENGLES_V2
-- Performing Test HAVE_OPENGLES_V2 - Success
-- Checking for modules 'wayland-client>=1.18;wayland-egl;wayland-cursor;egl;xkbcommon>=0.5.0'
--   Found wayland-client, version 1.23.1
--   Found wayland-egl, version 18.1.0
--   Found wayland-cursor, version 1.23.1
--   Found egl, version 1.5
--   Found xkbcommon, version 1.9.2
-- dynamic libwayland-client -> libwayland-client.so.0
-- dynamic libwayland-egl -> libwayland-egl.so.1
-- dynamic libwayland-cursor -> libwayland-cursor.so.0
-- dynamic libxkbcommon -> libxkbcommon.so.0
-- Checking for module 'libdecor-0'
--   Found libdecor-0, version 0.2.2
-- dynamic libdecor-0 -> libdecor-0.so.0
-- Performing Test HAVE_LINUX_INPUT_H
-- Performing Test HAVE_LINUX_INPUT_H - Success
-- Performing Test HAVE_INPUT_KD
-- Performing Test HAVE_INPUT_KD - Success
-- Performing Test HAVE_LINUX_VIDEODEV2_H
-- Performing Test HAVE_LINUX_VIDEODEV2_H - Success
-- Looking for libudev.h
-- Looking for libudev.h - found
-- dynamic libudev -> libudev.so.1
-- Looking for sys/inotify.h
-- Looking for sys/inotify.h - found
-- Looking for inotify_init
-- Looking for inotify_init - found
-- Looking for inotify_init1
-- Looking for inotify_init1 - found
-- Checking for one of the modules 'dbus-1;dbus'
-- Checking for one of the modules 'ibus-1.0;ibus'
-- Checking for one of the modules 'liburing-ffi'
-- Checking for module 'libusb-1.0>=1.0.16'
--   Package 'libusb-1.0' not found
-- Could NOT find LibUSB (missing: LibUSB_LIBRARY LibUSB_INCLUDE_PATH) (found version "LibUSB_VERSION-NOTFOUND")
-- Performing Test HAVE_PTHREADS
-- Performing Test HAVE_PTHREADS - Success
-- Performing Test HAVE_RECURSIVE_MUTEXES
-- Performing Test HAVE_RECURSIVE_MUTEXES - Success
-- Performing Test HAVE_PTHREADS_SEM
-- Performing Test HAVE_PTHREADS_SEM - Success
-- Performing Test COMPILER_HAS_SEM_TIMEDWAIT
-- Performing Test COMPILER_HAS_SEM_TIMEDWAIT - Success
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_np.h
-- Looking for pthread_np.h - not found
-- Performing Test HAVE_PTHREAD_SETNAME_NP
-- Performing Test HAVE_PTHREAD_SETNAME_NP - Success
-- Looking for clock_gettime in c
-- Looking for clock_gettime in c - found
-- Performing Test HAVE_POSIX_SPAWN
-- Performing Test HAVE_POSIX_SPAWN - Success
-- Found Perl: /usr/bin/perl (found version "5.40.2")
-- 
-- SDL3 was configured with the following options:
-- 
-- Platform: Linux-6.14.5-1-default
-- 64-bit:   TRUE
-- Compiler: /usr/bin/cc
-- Revision: SDL3-3.3.0-release-3.2.6-572-g4cd889cb1
-- Vendor:   
-- 
-- Subsystems:
--   Audio:    ON
--   Video:    ON
--   GPU:      ON
--   Render:   ON
--   Camera:   ON
--   Joystick: ON
--   Haptic:   ON
--   Hidapi:   ON
--   Power:    ON
--   Sensor:   ON
--   Dialog:   ON
-- 
-- Options:
--   SDL_ALSA                    (Wanted: ON): ON
--   SDL_ALSA_SHARED             (Wanted: OFF): OFF
--   SDL_ALTIVEC                 (Wanted: OFF): OFF
--   SDL_ARMNEON                 (Wanted: OFF): OFF
--   SDL_ASAN                    (Wanted: OFF): OFF
--   SDL_ASSEMBLY                (Wanted: ON): ON
--   SDL_ASSERTIONS              (Wanted: auto): auto
--   SDL_AVX                     (Wanted: ON): ON
--   SDL_AVX2                    (Wanted: ON): ON
--   SDL_AVX512F                 (Wanted: ON): ON
--   SDL_BACKGROUNDING_SIGNAL    (Wanted: OFF): OFF
--   SDL_CCACHE                  (Wanted: OFF): OFF
--   SDL_CLANG_TIDY              (Wanted: OFF): OFF
--   SDL_CLOCK_GETTIME           (Wanted: ON): ON
--   SDL_COCOA                   (Wanted: OFF): OFF
--   SDL_DBUS                    (Wanted: ON): ON
--   SDL_DEPS_SHARED             (Wanted: ON): OFF
--   SDL_DIRECTX                 (Wanted: OFF): OFF
--   SDL_DISKAUDIO               (Wanted: ON): ON
--   SDL_DUMMYAUDIO              (Wanted: ON): ON
--   SDL_DUMMYCAMERA             (Wanted: ON): ON
--   SDL_DUMMYVIDEO              (Wanted: ON): ON
--   SDL_EXAMPLES                (Wanted: OFF): OFF
--   SDL_EXAMPLES_LINK_SHARED    (Wanted: ON): OFF
--   SDL_FOREGROUNDING_SIGNAL    (Wanted: OFF): OFF
--   SDL_GCC_ATOMICS             (Wanted: ON): ON
--   SDL_GPU_DXVK                (Wanted: OFF): OFF
--   SDL_HIDAPI                  (Wanted: ON): ON
--   SDL_HIDAPI_JOYSTICK         (Wanted: ON): ON
--   SDL_HIDAPI_LIBUSB           (Wanted: ON): OFF
--   SDL_HIDAPI_LIBUSB_SHARED    (Wanted: ON): OFF
--   SDL_IBUS                    (Wanted: ON): ON
--   SDL_INSTALL                 (Wanted: ON): OFF
--   SDL_INSTALL_TESTS           (Wanted: OFF): OFF
--   SDL_JACK                    (Wanted: ON): ON
--   SDL_JACK_SHARED             (Wanted: ON): ON
--   SDL_KMSDRM                  (Wanted: ON): ON
--   SDL_KMSDRM_SHARED           (Wanted: OFF): OFF
--   SDL_LASX                    (Wanted: OFF): OFF
--   SDL_LIBC                    (Wanted: ON): ON
--   SDL_LIBICONV                (Wanted: OFF): OFF
--   SDL_LIBUDEV                 (Wanted: ON): ON
--   SDL_LIBURING                (Wanted: ON): OFF
--   SDL_LSX                     (Wanted: OFF): OFF
--   SDL_METAL                   (Wanted: OFF): OFF
--   SDL_MMX                     (Wanted: ON): ON
--   SDL_OFFSCREEN               (Wanted: ON): ON
--   SDL_OPENGL                  (Wanted: ON): ON
--   SDL_OPENGLES                (Wanted: ON): ON
--   SDL_OPENVR                  (Wanted: OFF): OFF
--   SDL_OSS                     (Wanted: OFF): OFF
--   SDL_PIPEWIRE                (Wanted: ON): ON
--   SDL_PIPEWIRE_SHARED         (Wanted: OFF): OFF
--   SDL_PTHREADS                (Wanted: ON): ON
--   SDL_PTHREADS_SEM            (Wanted: ON): ON
--   SDL_PULSEAUDIO              (Wanted: ON): ON
--   SDL_PULSEAUDIO_SHARED       (Wanted: OFF): OFF
--   SDL_RENDER_D3D              (Wanted: OFF): OFF
--   SDL_RENDER_D3D11            (Wanted: OFF): OFF
--   SDL_RENDER_D3D12            (Wanted: OFF): OFF
--   SDL_RENDER_GPU              (Wanted: ON): ON
--   SDL_RENDER_METAL            (Wanted: ON): OFF
--   SDL_RENDER_VULKAN           (Wanted: ON): ON
--   SDL_ROCKCHIP                (Wanted: OFF): OFF
--   SDL_RPATH                   (Wanted: OFF): OFF
--   SDL_RPI                     (Wanted: OFF): OFF
--   SDL_SNDIO                   (Wanted: ON): OFF
--   SDL_SNDIO_SHARED            (Wanted: ON): OFF
--   SDL_SSE                     (Wanted: ON): ON
--   SDL_SSE2                    (Wanted: ON): ON
--   SDL_SSE3                    (Wanted: OFF): OFF
--   SDL_SSE4_1                  (Wanted: ON): ON
--   SDL_SSE4_2                  (Wanted: ON): ON
--   SDL_SYSTEM_ICONV            (Wanted: ON): ON
--   SDL_TESTS                   (Wanted: OFF): OFF
--   SDL_TESTS_LINK_SHARED       (Wanted: ON): OFF
--   SDL_UNINSTALL               (Wanted: ON): OFF
--   SDL_VIRTUAL_JOYSTICK        (Wanted: ON): ON
--   SDL_VIVANTE                 (Wanted: OFF): OFF
--   SDL_VULKAN                  (Wanted: ON): ON
--   SDL_WASAPI                  (Wanted: OFF): OFF
--   SDL_WAYLAND                 (Wanted: ON): ON
--   SDL_WAYLAND_LIBDECOR        (Wanted: ON): ON
--   SDL_WAYLAND_LIBDECOR_SHARED (Wanted: OFF): OFF
--   SDL_WAYLAND_SHARED          (Wanted: OFF): OFF
--   SDL_X11                     (Wanted: ON): ON
--   SDL_X11_SHARED              (Wanted: OFF): OFF
--   SDL_X11_XCURSOR             (Wanted: ON): ON
--   SDL_X11_XDBE                (Wanted: ON): ON
--   SDL_X11_XFIXES              (Wanted: ON): ON
--   SDL_X11_XINPUT              (Wanted: ON): ON
--   SDL_X11_XRANDR              (Wanted: ON): ON
--   SDL_X11_XSCRNSAVER          (Wanted: ON): ON
--   SDL_X11_XSHAPE              (Wanted: ON): ON
--   SDL_X11_XSYNC               (Wanted: ON): ON
--   SDL_X11_XTEST               (Wanted: ON): ON
--   SDL_XINPUT                  (Wanted: OFF): OFF
-- 
--  Build Shared Library: ON
--  Build Static Library: OFF
-- 
-- If something was not detected, although the libraries
-- were installed, then make sure you have set the
-- CMAKE_C_FLAGS and CMAKE_PREFIX_PATH CMake variables correctly.
-- 
-- Configuring done (25.4s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_STATIC_LIBS
    CMAKE_CXX_FLAGS
    CMAKE_Fortran_FLAGS
    CMAKE_INSTALL_DO_STRIP
    CMAKE_MODULES_INSTALL_DIR
    INCLUDE_INSTALL_DIR
    LIB_INSTALL_DIR
    LIB_SUFFIX
    SDL_3DNOW
    SDL_LIBSAMPLERATE_SHARED
    SDL_STATIC_PIC
    SHARE_INSTALL_PREFIX
    SYSCONF_INSTALL_DIR


-- Build files have been written to: /home/ej/code/SDL
```

</details>

make output:

```
[ 62%] Building C object CMakeFiles/SDL3-shared.dir/src/video/x11/SDL_x11dyn.c.o
[ 63%] Building C object CMakeFiles/SDL3-shared.dir/src/video/x11/SDL_x11events.c.o
/home/ej/code/SDL/src/video/x11/SDL_x11sym.h: In function ‘SDL_X11_LoadSymbols’:
/home/ej/code/SDL/src/video/x11/SDL_x11sym.h:187:21: error: ‘XTestQueryExtension’ undeclared (first use in this function); did you mean ‘XGEQueryExtension’?
  187 | SDL_X11_SYM(Status, XTestQueryExtension, (Display* a, int* b, int* c))
      |                     ^~~~~~~~~~~~~~~~~~~
/home/ej/code/SDL/src/video/x11/SDL_x11dyn.c:192:67: note: in definition of macro ‘SDL_X11_SYM’
  192 | #define SDL_X11_SYM(rc, fn, params) X11_##fn = (SDL_DYNX11FN_##fn)fn;
      |                                                                   ^~
/home/ej/code/SDL/src/video/x11/SDL_x11sym.h:187:21: note: each undeclared identifier is reported only once for each function it appears in
  187 | SDL_X11_SYM(Status, XTestQueryExtension, (Display* a, int* b, int* c))
      |                     ^~~~~~~~~~~~~~~~~~~
/home/ej/code/SDL/src/video/x11/SDL_x11dyn.c:192:67: note: in definition of macro ‘SDL_X11_SYM’
  192 | #define SDL_X11_SYM(rc, fn, params) X11_##fn = (SDL_DYNX11FN_##fn)fn;
      |                                                                   ^~
/home/ej/code/SDL/src/video/x11/SDL_x11sym.h:188:18: error: ‘XTestFakeMotionEvent’ undeclared (first use in this function); did you mean ‘X11_XTestFakeMotionEvent’?
  188 | SDL_X11_SYM(int, XTestFakeMotionEvent, (Display* a, int b, int c, int d, unsigned long e))
      |                  ^~~~~~~~~~~~~~~~~~~~
/home/ej/code/SDL/src/video/x11/SDL_x11dyn.c:192:67: note: in definition of macro ‘SDL_X11_SYM’
  192 | #define SDL_X11_SYM(rc, fn, params) X11_##fn = (SDL_DYNX11FN_##fn)fn;
      |                                                                   ^~
[ 63%] Building C object CMakeFiles/SDL3-shared.dir/src/video/x11/SDL_x11framebuffer.c.o
make[2]: *** [CMakeFiles/SDL3-shared.dir/build.make:2654: CMakeFiles/SDL3-shared.dir/src/video/x11/SDL_x11dyn.c.o] Error 1
```

--- a/src/video/x11/SDL_x11dyn.h
+++ b/src/video/x11/SDL_x11dyn.h
@@ -71,6 +71,9 @@
 #ifdef SDL_VIDEO_DRIVER_X11_XSHAPE
 #include <X11/extensions/shape.h>
 #endif
+#ifdef SDL_VIDEO_DRIVER_X11_XTEST
+#include <X11/extensions/XTest.h>
+#endif
 
 #ifdef __cplusplus
 extern "C" {

